rpmlint/checks/DocCheck.py

```diff
@@ -52,7 +51,7 @@ def _check_doc_file_dependencies(self, pkg):
         Check if docfiles create additional dependencies in the package and
         print a warning if so.
         """
-        files = pkg.files()
+        files = pkg.files
```

Comment:
does this need separate local variable?

Reply:
i left local variables in places where they were used in more than 2 cases